packages/graphql-server-module-graphiql/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "graphql-server-module-graphiql",
-  "version": "0.4.4",
+  "version": "0.6.1",
```

Comment:
@DxCx do you mean it should be updated to `0.6.0`?
because otherwise the build here doesn't pass...
so we can merge it like so?
and then merge this https://github.com/apollographql/graphql-server/pull/334